packages/manager/modules/pci/src/projects/project/storages/databases/database/namespaces/edit/edit.html

```diff
@@ -0,0 +1,262 @@
+                    placeholder="{{:: 'pci_databases_namespaces_edit_name_placeholder' | translate }}"
+                    disabled
+                    required
+                />
+            </oui-field>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col">
+            <oui-field>
+                <label class="oui-field__label">
+                    <span
+                        data-translate="{{:: 'pci_databases_namespaces_edit_type_label' | translate }}"
+                    ></span>
+                    <small class="text-danger">*</small>
+                </label>
+                <button
+                    type="button"
+                    class="oui-popover-button"
+                    aria-label="Help"
+                    oui-popover="{{:: 'pci_databases_namespaces_edit_type_popover' | translate }}"
+                    oui-popover-id="popover-name"
+                    oui-popover-placement="right"
+                ></button>
+                <oui-radio-group model="$ctrl.model.type">
+                    <oui-radio inline value="'aggregated'" disabled>
+                        <span>Aggregated</span>
+                    </oui-radio>
+                    <oui-radio inline value="'unaggregated'" disabled>
+                        <span>Unaggregated</span>
+                    </oui-radio>
+                </oui-radio-group>
+            </oui-field>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col">
+            <oui-field>
+                <label class="oui-field__label">
+                    <span
+                        data-translate="{{:: 'pci_databases_namespaces_edit_retention_perdiod_duration_label' | translate }}"
+                    ></span>
+                    <small class="text-danger">*</small>
+                </label>
+                <button
+                    type="button"
+                    class="oui-popover-button"
+                    aria-label="Help"
+                    oui-popover="{{:: 'pci_databases_namespaces_edit_retention_perdiod_duration_popover' | translate }}"
+                    oui-popover-id="popover-name"
+                    oui-popover-placement="right"
+                ></button>
+                <input
+                    class="oui-input"
+                    type="text"
+                    id="retention_period"
+                    name="retention_period"
+                    placeholder="{{:: 'pci_databases_namespaces_edit_retention_perdiod_duration_placeholder' | translate }}"
+                    ng-model="$ctrl.model.retention.periodDuration"
+                    ng-model-options="{allowInvalid: true}"
+                    ng-pattern="$ctrl.FORM_RULES.periodDuration.pattern"
+                    ng-maxlength="$ctrl.FORM_RULES.periodDuration.max"
+                    required
+                />
+            </oui-field>
+        </div>
+        <div class="col" ng-if="$ctrl.model.type === 'aggregated'">
```

Comment:
**suggestion:** Can be maintained as a constant

Reply:
Done here: https://github.com/ovh/manager/pull/6414/commits/1580d8b5f3f673c8bae631f326c46fb4c2539c0c